Golang Tutorial is broken

The example controller code makes reference to a memcached.spec.port field, which doesn't exist in the spec and status objects that you copy+paste from the tutorial docs.

The tutorial is way more broken. The `operator-sdk init` command ends with:

```
FATA[0000] failed to initialize project: unable to run pre-scaffold tasks of "base.go.kubebuilder.io/v3": go version 'go1.18.9' is incompatible because 'plugin requires go1.19 <= version < go2.0alpha1'. You can skip this check using the --skip-go-version-check flag 
```
The Tutorial specifies that 1.18 is required. 

With go 1.19.4 (latest atm) the previous command does not fail, but later I get:
```
$ make docker-build docker-push IMG="example.com/memcached-operator:v0.0.1"
mkdir -p /Users/michalmedvecky/Documents/go/memcached-operator/bin
test -s /Users/michalmedvecky/Documents/go/memcached-operator/bin/controller-gen || GOBIN=/Users/michalmedvecky/Documents/go/memcached-operator/bin go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.10.0
/Users/michalmedvecky/Documents/go/memcached-operator/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
/Users/michalmedvecky/Documents/go/memcached-operator/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
go fmt ./...
go vet ./...
test -s /Users/michalmedvecky/Documents/go/memcached-operator/bin/setup-envtest || GOBIN=/Users/michalmedvecky/Documents/go/memcached-operator/bin go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
go: sigs.k8s.io/controller-runtime/tools/setup-envtest@latest: module sigs.k8s.io/controller-runtime@latest found (v0.13.1), but does not contain package sigs.k8s.io/controller-runtime/tools/setup-envtest
make: *** [/Users/michalmedvecky/Documents/go/memcached-operator/bin/setup-envtest] Error 1
```

--- a/website/content/en/docs/building-operators/golang/tutorial.md
+++ b/website/content/en/docs/building-operators/golang/tutorial.md
@@ -111,12 +111,20 @@ Define the API for the Memcached Custom Resource(CR) by modifying the Go type de
 type MemcachedSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
-	// Size defines the number of Memcached instances
+
 	// The following markers will use OpenAPI v3 schema to validate the value
+	// More info: https://book.kubebuilder.io/reference/markers/crd-validation.html
 	// +kubebuilder:validation:Minimum=1
 	// +kubebuilder:validation:Maximum=3
 	// +kubebuilder:validation:ExclusiveMaximum=false
+
+	// Size defines the number of Memcached instances
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	Size int32 `json:"size,omitempty"`
+
+	// Port defines the port that will be used to init the container with the image
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	ContainerPort int32 `json:"containerPort,omitempty"`
 }
 
 // MemcachedStatus defines the observed state of Memcached
@@ -128,6 +136,10 @@ type MemcachedStatus struct {
 	// condition types may define expected values and meanings for this field, and whether the values
 	// are considered a guaranteed API.
 	// Memcached.status.conditions.Message is a human readable message indicating details about the transition.
+	// For further information see: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties
+
+	// Conditions store the status conditions of the Memcached instances
+	// +operator-sdk:csv:customresourcedefinitions:type=status
 	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
 }
 ```

